### DISCUSSION TITLE
Faust Seo Meta problem and Yoast Seo supporting

### DISCUSSION THREAD

**Discussion by nikoids** [wpengine/faustjs#1683] [Ideas]
I tested your projects created on Faust, including your original blog, everyone has problems with SEO
When checking the site for meta everything is missing
can you provide a guide on how I can add yoast seo and create meta for posts, pages, categories for everything in general?
Thank you

**Comment by theodesp** [comment]
Hey @nikoids thank you for the feedback. 

AFIK Yoast support in wp-graphql is provided with this plugin https://github.com/ashhitch/wp-graphql-yoast-seo

We should definitely create a guide on how to leverage this plugin for sure since we don't have anything in our docs right now.

Are you also experiencing any issues with [next.js Head](https://nextjs.org/docs/pages/api-reference/components/head) component?

**Comment by theodesp** [comment]
Hey @nikoids I will move this to the conversation section.
